src/main/kotlin/com/openlattice/transporter/services/TransporterConfiguration.kt

```diff
@@ -0,0 +1,16 @@
+package com.openlattice.transporter.services
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.kryptnostic.rhizome.configuration.annotation.ReloadableConfiguration
+import java.util.*
+
+@ReloadableConfiguration(uri="transporter.yaml")
+data class TransporterConfiguration(
+        @JsonProperty val server: Properties,
+        @JsonProperty val ssl: Boolean = true,
+        @JsonProperty val once: Boolean = true
```

Comment:
what does `once` affect?

Reply:
was a way for me to run the full copy once and then exit. I'll remove it.